cli/image-scanning-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,68 @@
+[id="image-scanning-by-using-the-roxctl-cli"]
+:_mod-docs-content-type: ASSEMBLY
+= Image scanning by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: image-scanning-by-using-the-roxctl-cli
+
+toc::[]
+
+[role="_abstract"]
+You can scan images stored in image registries, including cluster local registries such as the {ocp} integrated image registry by using the `roxctl` CLI.
+
+[id="scanning-images-in-a-remote-cluster_{context}"]
+== Scanning images using a remote cluster
+
+By specifying the appropriate cluster in the delegated scanning configuration or the cluster parameter described in the following procedure, you can scan images from cluster local registries by using a remote cluster.
```

Comment:
This sentence need something. It's not clear to me what is being specified where. Is either of these right?

> By specifying the appropriate cluster... in the cluster parameter described in the following procedure, 

> By specifying ... the cluster parameter as described in the following procedure....

Do we need to show how to specify the appropriate cluster in the delegated scanning configuration?

Reply:
> This sentence need something. It's not clear to me what is being specified where. Is either of these right?
> 
> > By specifying the appropriate cluster... in the cluster parameter described in the following procedure,
> 
> > By specifying ... the cluster parameter as described in the following procedure....
> 
Please see comment: https://github.com/openshift/openshift-docs/pull/66481#discussion_r1374957553
I think we can make a slight adjustment to the content by adding "through the cluster parameter"
"By specifying the appropriate cluster in the delegated scanning configuration or **through** the cluster parameter described in the following procedure, you can scan images from cluster local registries by using a remote cluster."
 
> Do we need to show how to specify the appropriate cluster in the delegated scanning configuration?

I have already added a  cross-ref to the delegated image scanning section.